File: src/main/java/me/windyteam/kura/module/modules/misc/NoPacketKick.java
package me.windyteam.kura.module.modules.misc;

import me.windyteam.kura.module.Category;
import me.windyteam.kura.module.Module;
import me.windyteam.kura.mixin.client.MixinNetworkManager;

/**
 * @author 086
 * @see MixinNetworkManager
 */
@Module.Info(name = "NoPacketKick", category = Category.MISC, description = "Prevent large packets from kicking you")
public class NoPacketKick extends Module {
    public static NoPacketKick INSTANCE;

    public NoPacketKick() {
        NoPacketKick.INSTANCE = this;
    }

    public static boolean isEnable() {
        return NoPacketKick.INSTANCE.isEnabled();
    }
}

File: src/main/java/me/windyteam/kura/module/modules/misc/NoPacketKick.kt
package me.windyteam.kura.module.modules.misc

import me.windyteam.kura.module.Category
import me.windyteam.kura.module.Module

@Module.Info(name = "NoPacketKick", category = Category.MISC, description = "Prevent large packets from kicking you")
class NoPacketKick : Module() {
    init {
        INSTANCE = this
    }

    companion object {
        var INSTANCE: NoPacketKick? = null
    }
}